curriculum/challenges/english/01-responsive-web-design/css-piano/part-004.md

```diff
@@ -0,0 +1,70 @@
+---
+id: 612e77aba7ca691f598feb02
+title: Part 4
+challengeType: 0
+dashedName: part-4
+---
+
+# --description--
+
+You need to optimize the page to be responsive for small viewports, such as tablets and mobile devices. Within your `head` element, add a `meta` tag with the `name` attribute set to `viewport`, and the `content` attribute set to `width=device-width, initial-scale=1.0`.
```

Comment:
**Note**: This topic has previously been covered. We should meet to go through all the projects, and find more things to add to the initial boilerplate, or include more instructions like:
> Add a `meta` tag to optimize your page for mobile devices